lib/test/index.html

```diff
@@ -1,21 +0,0 @@
-<!DOCTYPE html>
-<html>
-<head>
-  <meta charset="utf-8">
-  <title>Ramda Tests</title>
-  <link rel="stylesheet" href="mocha.css" />
-</head>
-<body>
-  <h1>Tests for ramda</h1>
-  <div id="mocha"></div>
-  <script src="lib/test/expect.js"></script>
-  <script src="mocha.js"></script>
-  <script src="assert-shim.js"></script>
-  <!--[if lt IE 9]>
-    <script src="../lib/sauce/ie8-shim.js"></script>
-  <![endif]-->
-  <script>mocha.setup('bdd')</script>
-  <script src="bundle.js"></script>
-  <script src="../sauce/mocharun.js"></script>
-</body>
-</html>
```

Comment:
While this whole change is clearly moving in the right direction, I've got to admit to some sadness at seeing this file go.  This was how I started the library, long before it was called Ramda, a few tests that ran in Mocha and in the browser.  And if I had problems, I would debug it in Chrome or Firefox in the browser.  This existed before almost any of the other infrastructure of the library.

So :+1: , but :cry: